docs/content/contribute/how-to-contribute/contributing-to-discussions.adoc

```diff
@@ -0,0 +1,43 @@
+
+---
+title: "Contributing to discussions"
+weight: 1
+description: >
+  How to contribute to discussions & issues
+---
+
+The https://github.com/odu-cga-cubesat/flight-software/discussions[Discussions] and https://github.com/odu-cga-cubesat/flight-software/issues[Issues] sections help fascilitate invaluable feedback and discourse in the development and maintainability of the ODU/CGA CubeSat flight software.
```

Comment:
"fascilitate" misspelled.

Reply:
resolving in https://github.com/ODU-CGA-CubeSat/mission-conops/pull/28/commits/5a88a8cc9653ad850cc0ae965e81305b807d2e92